generators/chipyard/src/main/scala/Subsystem.scala

```diff
@@ -77,8 +77,6 @@ class ChipyardSubsystem(implicit p: Parameters) extends BaseSubsystem
 class ChipyardSubsystemModuleImp[+L <: ChipyardSubsystem](_outer: L) extends BaseSubsystemModuleImp(_outer)
   with HasTilesModuleImp
 {
-  // create file with core params
-  ElaborationArtefacts.add("""core.config""", outer.tiles.map(x => x.module.toString).mkString("\n"))
```

Comment:
I guess this becomes too unwieldy with tons of cores? I personally liked this for quick readability of params.

Reply:
Its because trying to access tile.module in a cloned LazyModule causes problems. There needs to be a differnt way to report core params.